### PR TITLE
[otp_ctrl] Sign v2.0.0 off at D3

### LINKED ISSUE #22479: [otp_ctrl] D3 Signoff
The previous version of otp_ctrl has been signed off at D3, but we need to go through the signoff checklist for the current version.

### PR DESCRIPTION
**Pull request by andreaskurth**
See issue #22479 for details. This closes #22479.

### CODE REVIEW

**Comment by andreaskurth** [comment]
CHANGE AUTHORIZED: hw/ip/otp_ctrl/data/otp_ctrl.hjson

**Review by vogelpi — APPROVED** [review]
Thanks @andreaskurth. I've reviewed the sign-off issue and this PR and this all looks great to me!

File: hw/ip/otp_ctrl/doc/checklist.md [hunk 1]
```diff
@@ -109,8 +109,8 @@ Review        | [REVIEW_RTL][]          | Done        |
 Review        | [REVIEW_DELETED_FF][]   | Waived      | No block-level flow available - waived to top-level signoff.
 Review        | [REVIEW_SW_CHANGE][]    | Done        |
 Review        | [REVIEW_SW_ERRATA][]    | Done        |
-Review        | Reviewer(s)             | Done        | msf@ gac@ chencindy@ awill@
-Review        | Signoff date            | Done        | 2022-06-27
+Review        | Reviewer(s)             | Done        | adk@ msf@ gac@ chencindy@ awill@
```
[the remark this answers is not in the document]
Comment (andreaskurth) [reply hw/ip/otp_ctrl/doc/checklist.md:112]: OK, so in this case only you and me, right?

[the remark this answers is not in the document]
Comment (andreaskurth) [reply hw/ip/otp_ctrl/doc/checklist.md]: OK, done

**Comment by andreaskurth** [comment]
CI lint checks passed, and the failing CI checks cannot be caused by this PR. Merging.

**Pull request merged**